docs/styling-engines.md

```diff
@@ -0,0 +1,25 @@
+# 🕴️ Styling engines
```

Comment:
Is this documentation intended for the user of UI Toolkit or for the development of UI Toolkit? It seems to be a mix.